Modules/LinearBoltzmannSolver/IterativeMethods/lbs_classicrichardson.cc

```diff
@@ -80,7 +80,7 @@ void LinearBoltzmann::Solver::
     pw_change_prev = pw_change;
 
     if (k==0) rho = 0.0;
-    if (pw_change<std::max(groupset->residual_tolerance*rho,1.0e-10))
+    if (pw_change<std::max(groupset->residual_tolerance*(1.0-rho),1.0e-10))
```

Comment:
Someone should check me on this, but I'm pretty sure we should be multiplying by (1.0-rho) not rho. If we multiply by rho, then as rho->1, we converge to residual_tolerance, but I think the point of this is to avoid false convergence and so we want to converge to a smaller tolerance as rho->1 right?

Reply:
Mmmmm. It's been a while since I've worked with Classic Richardson but your argument seems to check out. Go ahead with this change.